Adyen.EcommLibrary/HttpClient/HttpURLConnectionClient.cs

```diff
@@ -60,14 +60,14 @@ public string Request(string endpoint, string json, Config config, bool isApiKey
         /// <param name="config"></param>
         /// <param name="isApiKeyRequired"></param>
         /// <returns>Task<></returns>
-        public async Task<string> RequestAsync(string endpoint, string json, Config config, bool isApiKeyRequired)
+        public async Task<string> RequestAsync(string endpoint, string json, Config config, bool isApiKeyRequired, string idempotencyKey = null)
```

Comment:
This approach is not backward compatible because RequestAsync with 3 parameters will not be supported and some merchants may not want to use idempotency key.